src/components/UI/common/FooterBackgroundImage.tsx

```diff
@@ -14,7 +14,6 @@ export const FooterBackgroundImage: FC = () => {
           alt='People gathered around the Ethereum tree'
           layout='responsive'
           objectFit='cover'
-          quality={85}
```

Comment:
Didn't work as expected?

Reply:
I didn't notice any difference in this case with the default quality (75) and it was adding file size.